### ISSUE TITLE
Will not work even after new download

### ISSUE THREAD

**Issue by Alphatech81** [Dimencia/DU-Orbital-Hud#185]
Hi, Since the new update from the game it doesn't work, I downloaded your latest which was about 8 hours ago and I updated the config list in-game and made sure its keyboard only with control as per instructions listed but it's still not working, it only shows and is operating on original game HUD
A lot of people seem to have no problem with it but for some reason, probably my fault which I cannot work out, it's not working at all for me. any advice or tips are most appreciated.

**Comment by Alphatech81** [comment]
I didnt state keyboard  in the lua config
